src/graphql/error-map.ts

```diff
@@ -105,7 +106,21 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
         "Too many onchain addresses creation, please wait for a while and try again."
       return new TooManyRequestError({ message, logger: baseLogger })
 
+    case "RouteNotFoundError":
+      message = "Unable to find a route for payment."
+      return new RouteFindingError({ message, logger: baseLogger })
+
+    case "InsufficientBalanceForRoutingError":
+      message = "Insufficient balance for finding a route."
+      return new RouteFindingError({ message, logger: baseLogger })
+
+    case "UnknownRouteNotFoundError":
+      message = "Unknown error occurred when trying to find a route for payment."
+      return new RouteFindingError({ message, logger: baseLogger })
+
     case "UnknownLnInvoiceDecodeError":
+      // Consider using `error.message` somehow since lib returns semi-sensible details
```

Comment:
```suggestion
      // TODO: Consider using `error.message` somehow since lib returns semi-sensible details
```

Reply:
TODO and FIXME are 2 keywords that would be automatically highlighted in many IDE. so when there are things left to do in later PR it's always good to add the comment in code directly (typically better than adding a related issues that get lost eventually)